### PR TITLE
EOS-20595: Deployment failing while applying utils component

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise <yashodhan.pise@seagate.com>

### CODE REVIEW

File: srv/components/misc_pkgs/kafka/teardown.sls [hunk 1]
```diff
@@ -35,6 +35,14 @@ Remove zookeeper log directory:
   file.absent:
     - name: /var/log/zookeeper
 
+Remove accepted epoch:
```
Comment (sachinpunadikar) [inline srv/components/misc_pkgs/kafka/teardown.sls:38]: Better cleanup entire /var/lib/zookeeper

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Cleanup the entire directory as mentioned in comment

File: srv/components/misc_pkgs/kafka/teardown.sls (hunk 1 above)
Comment (ypise) [reply srv/components/misc_pkgs/kafka/teardown.sls:38]: @sachinpunadikar 
We are doing that already on line 32 above.
Good catch. We don't need this change. There's something else we need to check.

**Comment by ypise** [comment]
These changes have already been covered in the teardown.sls file line 32.
There is no need to deliver this PR.

**Pull request closed without merging**